rss_repository.py

```
@@ -0,0 +1,55 @@
+        row = self.cursor.fetchone()
+        return row[0] if row else None
+    
+    def insert_entries(self, entries):
+        sql = """
+            INSERT OR IGNORE INTO rss_entries (url, title, pub_date, summary, cover_url)
+            VALUES (:link, :title, :pub_date, :summary, :cover_link )
+            """
+        self.cursor.executemany(sql, entries)
+        self.connection.commit()
+
+    def delete_old_entries(self, lifetime=14):
```

Ruff (0.11.9)

Missing return type annotation for public function `delete_old_entries`

Add return type annotation: `None`

(ANN201)